Documentation/doc/Documentation/dependencies

```diff
@@ -57,7 +57,7 @@ Principal_component_analysis
 STL_Extension
 Skin_surface_3
 Snap_rounding_2
-Sweep_line_2
+Surface_sweep_2
```

Comment:
This change will require a redirect in the apache file of the doc so that latest will be correctly redirected.